05-conclusiones.qmd

```diff
@@ -5,18 +5,23 @@ editor: visual
+Hypothesis $H_2$ proposed that students from higher socioeconomic status families would show stronger preferences for market justice. However, the results do not support this hypothesis since they indicate that this relationship is not as straightforward as expected. Although students from families with higher educational levels and more economic resources tend to prefer less market justice, this relationship is not statistically significant. This might be due to the influence of other contextual and cultural factors that modulate attitudes toward market justice [@kohn_social_1963; @kohn_class_1969; @almas_fairness_2017]. Notably, socioeconomic status measured by parental educational level and the number of books at home do not explain variations in market justice preferences, suggesting that other variables, such as market justice preferences of parents, could play an essential role in the attitudinal socialization process. This aligns with the notion that beliefs and attitudes toward social justice are complex and influenced by multiple factors beyond just economic resources.
 
-Hypothesis Hx held that students from schools with higher school achievement would show stronger preferences for market justice. Contrary to that hypothesis, the results show that students from schools with greater achievement in the SIMCE test tend to prefer less market justice.
+Hypothesis $H_3$ held that students from schools with higher socioeconomic status would show stronger preferences for market justice. The results did not confirm this hypothesis since although students from high-status schools tend to prefer more market justice, this result is not statistically significant. This finding, contrary to the existing literature [@jonsson_institutional_2015; @jost_attitudinal_2000], does not allow us to affirm the hypothesis about the role of the school environment, particularly socioeconomic segregation, in influencing attitudes towards market justice and the acceptance of inequalities. One possible explanation is the role of school-level variables not fully captured in our models, such as the school's climate or peer influence. Additionally, the non-significant results may suggest that the effect of school socioeconomic status on market justice preferences is correlated to other factors, such as individual meritocratic beliefs.
+
+Hypothesis $H_x$ [Falta hipótesis school achievement] held that students from schools with higher school achievement would show stronger preferences for market justice. Contrary to that hypothesis, the results show that students from schools with greater achievement in the SIMCE test tend to prefer less market justice. This unexpected finding could be due to the nuanced relationship between academic success and social attitudes. Students in high-achieving schools may be more exposed to critical discussions about inequality and social justice, leading to a rejection of market-based justifications for access to social services.
+
```

Comment:
Por favor @Kevin-carrasco ojo con las frases, por ejemplo: "This unexpected finding could be due to the nuanced relationship between academic success and social attitudes." Esto en realidad es lo mismo que no decir nada, demasiado general (muy tipo GPT).